docs/API.md

```diff
@@ -73,15 +73,41 @@ This sections describes a number of data types used in this API. Each data type
 is based on a JSON data type, but additionally imposes its own restrictions.
 
 
+### API key
+
+Basic data type: string
+
+This parameter is a free-form string that represent the password of an authenticated account (see [*Privilege levels*](API.md#Privilege levels))
```

Comment:
Please update the link target to #privilege-levels. Since it's in the same document you don't have to specify API.md. Also, if you hover your mouse over rendered markdown heading on Github, a link symbol appears with a link containing the fragment for the heading itself.